PDFWriter/CharStringType2Interpreter.cpp

```diff
@@ -452,7 +452,8 @@ Byte* CharStringType2Interpreter::InterpretCallSubr(Byte* inProgramCounter)
 		return NULL;
 
 	aCharString = mImplementationHelper->GetLocalSubr(mOperandStack.back().IntegerValue);
-	mOperandStack.pop_back();
+	if(mOperandStack.size() > 0)
```

Comment:
this line doesn't make sense.

there's an earlier line in 451 that checks if (mOperandStack.size() < 1) and return null.

cant be that that line didnt cause a return but then later the size is 0.

consider testing your case against main...i only just put this check there now.

Reply:
I cherry picked from v4.6.1, this pull request does not make sense